.ci/scripts/test.sh

```diff
@@ -83,4 +83,4 @@ NODE_VERSION=${NODE_VERSION} docker-compose \
   --no-ansi \
   --log-level ERROR \
   -f ${DOCKER_FOLDER}/${DOCKER_COMPOSE_FILE} \
-  down -v
+  down -v --remove-orphans || true
```

Comment:
1. Adding `--remove-orphans` sounds reasonable to me.
2. I'm not sure I want to add the `|| true` yet because it could mask real issues, right?

Recall that I said it was crazy we were setting up so many docker containers for the "@hapi/hapi" TAV tests (e.g. https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-nodejs%2Fapm-agent-nodejs-mbp/detail/master/763/pipeline/1290)? I think I see the issue why. It is a fallback here:
https://github.com/elastic/apm-agent-nodejs/blob/c94c799502dc4df2b09e513275337d5c98ee0d4e/.ci/scripts/test.sh#L9-L38

At least for that particular Jenkins build, the `error while removing network: network docker_default id ... has active endpoints` failures were for the steps testing "@hapi/hapi" and "@elastic/elasticsearch" which are missing from https://github.com/elastic/apm-agent-nodejs/blob/c94c799502dc4df2b09e513275337d5c98ee0d4e/.ci/scripts/test.sh#L10

I think we should update that `case ...` to include those two.
I also think we should consider warning or *erroring* in the `*)` case: https://github.com/elastic/apm-agent-nodejs/blob/c94c799502dc4df2b09e513275337d5c98ee0d4e/.ci/scripts/test.sh#L37

It may be that `docker-compose down ...` here is eventually still an issue, but I don't want to `|| true` it out yet.